pyartcd/pyartcd/pipelines/comment_to_pr.py

```diff
@@ -1,6 +1,8 @@
+import asyncio
 import pprint
 import sys
 import re
+import aiohttp
 import requests
```

Comment:
`requests`? I'm using it [here](https://github.com/ashwindasr/aos-cd-jobs/blob/c4fb2709114351e0a47ecf22adcff1057e8581a9/pyartcd/pyartcd/pipelines/comment_to_pr.py#L267) for the first data fetch from the DB

Reply:
Right. We should get rid of it and use `aiohttp` everywhere for consistency. Then, `_builds_from_job` would become async. This might or might not come in handy for the future, but once we decide to go async I'd say let's do it completely :)

Reply:
Got it! Fixed in https://github.com/openshift-eng/aos-cd-jobs/pull/3540/commits/2d2d68de8b04f6ec4f5b9cba8c2ba6a2a4b8b42c